bool use_odom
bool use_imu
bool use_laser
bool use_camera
bool use_battery
bool use_press